sig-network/charter.md

```diff
@@ -0,0 +1,43 @@
+# SIG Network
+
+## Scope
+
+SIG Network is owning everything related to KubeVirt networking.
+The main objective of the SIG is to connect the Virtual Machine to the primary and secondary pod networks,
+allowing migration and other advantages of virtualization in addition to maintaining services,
+port-forwarding, service meshes, and other Kubernetes capabilities.
+
+### In scope
+Some notable examples of features owned by SIG Network:
+- Masquerade, Bridge and Passt binding plugins for the primary network.
+- Bridge, SRIOV and MacVTap bindings for secondary networks.
+- VM networking lifecycle on: creation, migration, hotplug/unplug, restart, stop etc.
+
+#### Code, Binaries and Services
+- Networking related APIs on: VM, VMI, Instance types, Kubevirt CR, etc.
+- Core network binding infrastructure.
+- Network binding plugin infrastructure.
+- Examples of binding plugin CNIs and sidecars.
+- Virtual Machine/Virtual Machine Instance networking status management.
+- Domain networking configuration.
+- Unit and e2e tests.
+- Developer and user documentation around networking.
+- Help for onboarding new SIG Network members.
+- Grooming network related bugs.
+
+### Out of scope
+- Node networking, it is managed by [kubernetes-nmstate](https://github.com/nmstate/kubernetes-nmstate).
+- Host to Pod connectivity, it is managed by external CNIs.
+
+## Roles and Organization Management
+
+This sig follows adheres to the Roles and Organization Management outlined in [OARP]
+
+### Additional responsibilities of Chairs
```

Comment:
Did I miss where the "primary" repo was mentioned?

Reply:
I wasn't. Where do you think I should add it?

Reply:
Maybe in `Code, Binaries and Services` ?